cookbooks/aws-parallelcluster-computefleet/kitchen.computefleet-config.yml

```diff
@@ -0,0 +1,21 @@
+---
+verifier:
+  name: inspec
+  inspec_tests:
+    - cookbooks/aws-parallelcluster-computefleet/test
+
+suites:
+  - name: clusterstatusmgtd
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-computefleet::clusterstatusmgtd_config]
+    verifier:
+      controls:
+        - /tag:config_clusterstatusmgtd/
+    attributes:
+      dependencies:
+        - recipe:aws-parallelcluster-platform::users
+        - recipe:aws-parallelcluster-platform::directories
```

Comment:
Lately we tried to remove these dependencies making code units auto-consistent by creating the dependencies they needed wherever possible.

Reply:
directory creation moved into the recipe.